ethereum/api/src/integration-test/java/org/hyperledger/besu/ethereum/api/jsonrpc/methods/fork/frontier/EthCreateAccessListIntegrationTest.java

```diff
@@ -236,7 +236,12 @@ private void assertAccessListExpectedResult(
         new JsonRpcSuccessResponse(null, new CreateAccessListResult(accessList, gasUsed));
 
     final JsonRpcResponse response = method.response(request);
-    assertThat(response).usingRecursiveComparison().isEqualTo(expectedResponse);
+    assertThat(response)
```

Comment:
Is this required for the dependency upgrade somehow?

Reply:
it is required because with the update, `hashCode` fields are compared, and some tuweni classes compute the field lazily, so even if the object are equals, the `hashCode` field in the `expectedResponse` is `null`, so for those classes instead of a field by field comparison we use the `equals`